rcljava/src/main/cpp/org_ros2_rcljava_graph_NodeNameInfo.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "org_ros2_rcljava_graph_NodeNameInfo.h"
+
+#include <jni.h>
+#include <limits>
+
+#include "rcl/graph.h"
+#include "rcljava_common/exceptions.hpp"
+
+using rcljava_common::exceptions::rcljava_throw_exception;
+
+JNIEXPORT void JNICALL
+JNICALL Java_org_ros2_rcljava_graph_NodeName_nativeFromRCL(
+  JNIEnv * env, jobject self, jlong name_handle, jlong namespace_handle, jlong enclave_handle)
+{
+  const char * name = reinterpret_cast<const char *>(name_handle);
+  const char * n_namespace = reinterpret_cast<const char *>(namespace_handle);
+  const char * enclave = reinterpret_cast<const char *>(enclave_handle);
```

Comment:
Should we guard against null handles? I don't know how this native method is intended to be used within the context of this PR.

Reply:
Code was removed https://github.com/osrf/ros2_java/pull/31/commits/8734ea82b61fac64d1656169862cfc0577d6d879